src/test/scala/com/microsoft/hyperspace/index/rules/RuleUtilsTest.scala

```diff
@@ -167,6 +168,7 @@ class RuleUtilsTest extends HyperspaceRuleTestSuite {
           case _ =>
         }
       }
+       */
 
       {
         val optimizedPlan = spark.read.parquet(dataPath).queryExecution.optimizedPlan
```

Comment:
Don't you need to comment out the following line? ` verify(optimizedPlan, hybridScanEnabled = true, expectCandidateIndex = true)`? Is this also breaking when `hybridScanEnabled = false`?